source/frontend/src/components/common/mapFSM/MapStateMachineContext.tsx

```diff
@@ -384,20 +401,22 @@ export const MapStateMachineProvider: React.FC<React.PropsWithChildren<unknown>>
         showPopup: showPopup,
         isLoading: state.context.isLoading,
         mapSearchCriteria: state.context.searchCriteria,
+        advancedSearchCriteria: state.context.advancedSearchCriteria,
         mapFeatureData: state.context.mapFeatureData,
         filePropertyLocations: state.context.filePropertyLocations,
         pendingFitBounds: state.matches({ mapVisible: { mapRequest: 'pendingFitBounds' } }),
         requestedFitBounds: state.context.requestedFitBounds,
         isSelecting: state.matches({ mapVisible: { featureView: 'selecting' } }),
         isRepositioning: isRepositioning,
         selectingComponentId: state.context.selectingComponentId,
-        isFiltering: state.context.isFiltering,
```

Comment:
it is slightly harder to determine if we are filtering now, as in some ways we are "always" filtering. isFiltering is now true if we are using a non-default filter. Perhaps isFiltering could use a rename?